index.html

```diff
@@ -0,0 +1,182 @@
+            <h1 class="hero-h1">
+              Friends The Reunion <span>Super Fan's Conference</span>
+              <span>2023</span>
+            </h1>
+          </div>
+          <div class="welcome">
+            <p class="welcome-text">
+              Welcome to the Friends TV show cast reunion event! Join us for a
+              night full of surprises, special guests, and the iconic cast
+              members sharing their memories and behind-the-scenes stories.
+              Let's celebrate the magic of Friends together! Taking place in
+              Addis Ababa on April 2023.
+            </p>
+          </div>
+
+          <div class="date-location">
+            <h2>2023.04.15(MON) ~ 20(FRI)</h2>
+            <h3>@ Millennium convention center, Addis Ababa</h3>
+          </div>
+        </div>
+        <img
+          class="friends"
+          src="assets/images/friend-no-bg.png"
+          alt="Friends cast Group picture"
+        />
+      </section>
+    </section>
+    <!-- Mid Section -->
+    <section class="main-program" id="program">
+      <h2>Main Events</h2>
+      <ul class="events">
+        <li class="event">
+          <img src="assets/images/keynote.svg" alt="keynote icon" />
+          <p class="event-title">Stories</p>
+          <p class="event-desc">
+            The producers will share a stories never told before on what goes on
+            behind the scene.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/build.svg" alt="" />
+          <p class="event-title">Build</p>
+          <p class="event-desc">
+            Build competion of sets replica by top designers. Winners will get
+            big prizes.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/expo.svg" alt="Expo Icon" />
+          <p class="event-title">Expo</p>
+          <p class="event-desc">
+            A tour of the studio and showcase of unique items that make the show
+            unforgetable.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/forum.svg" alt="" />
+          <p class="event-title">Q&A</p>
+          <p class="event-desc">
+            You will get the chance to ask questions that were bugging you, all
+            cast on crew will be on stage to take your questions.
+          </p>
+        </li>
+        <li class="event">
+          <img src="assets/images/party.svg" alt="Party Icon" />
+          <p class="event-title">Party</p>
+          <p class="event-desc">
+            After party with the entire crew and producers, its gonna be hot and
+            you dont wann miss it.
+          </p>
+        </li>
+      </ul>
+      <button class="get-ticket">Get your ticket Here</button>
+    </section>
+    <!-- Guest section -->
+    <section class="guests" id="guests">
+      <h2>Featured Actors</h2>
+      <div class="guests-list"></div>
+    </section>
```

Comment:
Great job with the design. However, this section doesn't quite match the Behance design. I would recommend scaling down your image in height, so as to fit the text also.

| Your design  | Behance design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/224684412-67a1f28c-1ef3-40f8-841e-63a13381d08b.png) | ![image](https://user-images.githubusercontent.com/33268927/224684551-cd581c6f-6c79-4a91-9052-85185d86c662.png) |